.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "proto/opencensus-proto"]
+	path = proto/opencensus-proto
+	url = https://github.com/census-instrumentation/opencensus-proto
```

Comment:
Am not sure if we should be git submodule-ing stuff into Go as the convention is to use vendor. However this isn't a Go package plus using a git submodule will help us keep a separate git tree and update it when needed. Let's then add some instructions though for updating this submodule, if we go forward with it.

Reply:
I thought it would be okay in this case since the submodule is only required for "go generate", not when using the library.